apps/etterlatte-beregning-kafka/src/main/kotlin/no/nav/etterlatte/beregningkafka/MigreringHendelser.kt

```diff
@@ -43,25 +44,56 @@ internal class MigreringHendelser(rapidsConnection: RapidsConnection, private va
 
         beregningService.opprettBeregningsgrunnlag(behandlingId, tilGrunnlagDTO(packet.hendelseData))
 
-        runBlocking {
-            val beregning = beregningService.beregn(behandlingId).body<BeregningDTO>()
-            packet[BEREGNING_KEY] = beregning
-        }
+        val beregning =
+            runBlocking {
+                beregningService.beregn(behandlingId).body<BeregningDTO>()
+            }
+
+        val migreringRequest = objectMapper.treeToValue(packet[HENDELSE_DATA_KEY], MigreringRequest::class.java)
```

Comment:
Trur du kan seie `val migreringRequest = packet.hendelseData` her (og i det samme innser eg at det er eit litt dårleg namn...)

Reply:
Ah, nice! Fikset den for trygdetid også